BLSQ/iaso
# Polio 1330 vaccine stock modals (mock data)

modals fro FormA, Destruction and Incident reports

Related JIRA tickets :  POLIO-1330

## Self proofreading checklist


## Changes

- Add modal for each report type
- Form validation with yup is not implemented at this stage

## How to test

- With `SHOW_DEV_FEATURES` feature flag enabled on your local polio account
- Go to Vaccine Module > Stock Management > details icon button > Show variation
- Play around with the create button and edit icon button for each tab. The `body` and the expected request type are logged in the console

## Print screen / video



https://github.com/BLSQ/iaso/assets/38907762/0824ed36-156d-471a-bf5f-e5490b6aeba6
